### DISCUSSION TITLE
Possible heirlooms items added LEVELING PURPOSE ONLY!

### DISCUSSION THREAD

**Discussion by Neversong97** [chromiecraft/chromiecraft#4405] [Suggestions]
So, as tittle says I just would like to hear what other people think about adding heirlooms items for leveling and twink purposes ONLY!
To not make items take over as BIS on lvl 60 etc... They should STOP SCALING at lvl 50, and as a addition only people with level 60 character and above can get heirlooms for free on Chromiecraft website. (1 character per heirloom, so that means if u have x2 lvl 60 characters you can choose x2 heirlooms if you got 3x for example you can choose x3 heirlooms).

Reason is why am I suggesting this is because there are less people leveling and some quests can be pain if you dont get into a group with other players. Am just curious what other people think about this idea?
And am sure that people with already maxed character would like faster leveling that those items brings (max 20% increase xp)

PPC ofc,
Make sure you say Hi to Mez.

**Comment by heyitsbench** [comment]
Not sure this (stop scaling) idea is possible. 😄

**Reply by Neversong97** [reply]
if its not possible simply they disappear when you hit 50 for example or 55.

**Comment by Cooties-cc** [comment]
I don't think adding heirlooms will motivate ppl to level more characters. You either hate leveling, or don't mind it. TBC and Wrath already lowered the required exp to get to cap, as well as increasing the exp gained from quests. We have double exp every weekend. The main draw of the BoA items was the scaling factor in that you didn't have to worry about having a good gear prog path. There's nothing really wrong with that as it's nice to not have to worry about having to find weapon upgrades as a melee, etc. However I still think there are going to be further requests if BoA was allowed. These kinds of people are just never satisfied until it becomes instant cap w/ free gear. Then they quit cause they are bored. Many such cases.

I don't mind leveling at all, and have every class except DK to 60 currently. Mostly just as farming alts and maybe later a 2nd main. The average time it took (pre 2x exp) was around 2-3 days ingame played. That's basically nothing to hit level 60. With double exp weekends, that time could be down to 1 day or less played ingame. While I may be considered "exception to the rule", I honestly think that if people are unsatisfied with the current rate to hit level 60, they have a personal problem they need to deal with, and BoA gear or other increased exp methods are not going to fix that. 

Edit : 
![play time](https://user-images.githubusercontent.com/62584589/200921248-8ca462fb-534c-4a15-972a-8a4f8a1c0b92.png)
Here is a SS of each 60 character I have. Some are a little skewed due to playing during prev caps (hunter, priest, warrior mainly). Overall it doesn't take a lot of effort to hit 60, even as solo leveler which all my characters have been.

**Comment by Amrx2** [comment]
Hi Neversong, amending stats of items, changing scaling etc is not something we want to do. Especially with double xp weekends, RAF etc (and the fact we are regularly hitting around 700 players on now during weekends which over half are below 60), this isn't something we want to bring in for the time being.

Appreciate the suggestion, have a good day!
Construct
